arduino/finish_line/finish_line.ino

```diff
@@ -0,0 +1,92 @@
+#include <MQTT.h>
```

Comment:
Don't merge until you build this on CI as well

Reply:
> @Dreia where is this `finish_line.ino` sketch then? If it's source code it should be version controlled and built on CI for the very reason of avoiding such mistakes.

It is in our Arduino folder and it is version controlled. It has its own folder called finish_line containing vehicle_config as well to make it easy to drag and drop in the right place.

Reply:
> > @Dreia where is this `finish_line.ino` sketch then? If it's source code it should be version controlled and built on CI for the very reason of avoiding such mistakes.
> 
> It is in our Arduino folder and it is subjected to CI. It has its own folder called finish_line containing vehicle_config as well to make it easy to drag and drop in the right place.

Reply:
How is it "subjected to CI"? 🤔

Reply:
At least that is what I was told, I guess I was deceived. We'll look into it